### ISSUE TITLE
Compressed Output, incorrect SourceMap line numbers

### ISSUE THREAD

**Issue by tD3rk** [jasonsanjose/brackets-sass#110]
I've noticed when setting my output style to compressed, line numbers are incorrect when inspecting in browser (sourcemaps enabled). When setting output style to nested, numbers are correct. This is the case for me with a fairly vanilla setup, few partials being imported ahead of the lines in question that are off.

**Comment by carobe** [comment]
I have the same problem, but for me with style compressed or nested, the inspector never gives me the right number line.

**Comment by nickfmc** [comment]
I also receive this error on Compressed or Nested, I'm posting it on this ticket because I'm not sure the error occurred until I set the project to Compressed and then back to nested. V.2.0.1-109 Reinstalled, issue remains. Windows 8.1

**Comment by nicolo-ribaudo** [comment]
That is a libsass/node-sass bug, so can't be fixed in this extension :cry:

**Comment by nickfmc** [comment]
Do you have a link nicolo with info on this bug so when this ticket is closed and people find it they will have a resource?

**Comment by nicolo-ribaudo** [comment]
No, I just installed node-sass and compiled files from the command line.
